src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
 }
+$cell-size: 100px;
+$days:  "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
+$gap-size: 1px;
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Please, organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)